CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contribution Guide
+
+jargonLint aims to be accessible to all developers and writers, regardless of prior experience. However, we ask that you follow this guide in your contributions. This helps us support consistency and structure within the repository.
+
+## Simple Fixes
+
+There can be simple typos or edits to only a few lines that you want to correct.
+
+In this case, you can simply press the "edit" option in the upper page and GitHub should guide you through:
+1. Creating a fork
+2. Editing the fork
+3. Opening a pull request
+
+If the GitHub suggestions you should see on the upper area are not satisfactory, and you have limited experience, you can simply open an issue for your fix. 
+
+## Adding New Rules
+
+Steps to follow:
+
+1. Create a folder in the sources directory that carries the name of your rule without the ".yml" extension.
+2. Enter the (RuleName) folder.
+3. Create a "README.md" file. 
+    - This file should detail your motivation, methodology, notes, and sources. 
+    - Please use the template in the sources directory to write this file.
+4. Append the licenses from your sources in a file named "LICENSE.OLD"
+5. If useful, add any modified text that you think can be used in the future to this directory.
+6. Add your rule where the naming convention is "TitleCase.yml".
```

Comment:
This one is also known as PascalCase

Reply:
I know, but I thought it could be easier to suggest TitleCase since everyone knows that. Do you disagree?

Reply:
We could add it like that

TitleCase (aka. PascalCase)

Reply:
Just checked and TitleCase is a bit of a bastardization, I'll change it to PascalCase